package.json

```diff
@@ -94,5 +95,6 @@
     "zlib-sync": {
       "optional": true
     }
-  }
+  },
+  "packageManager": "pnpm@9.15.4+sha512.b2dc20e2fc72b3e18848459b37359a32064663e5627a51e4c74b2c29dd8e8e0491483c3abb40789cfd578bf362fb6ba8261b05f0387d76792ed6e23ea3b1b6a0"
```

Comment:
pnpm is a based tool but forcing it at a project level shouldn't be done in this PR, please remove this when done

Reply:
Whoops, didn't mean to leave that in